conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
@mrunalp thanks for the review. You're right, runc does expect take [`--bundle`](https://github.com/opencontainers/runc/blob/master/create.go#L28) as an argument.I am not sure the spec needs to be changed though, because there are already several implementations out there that implement the spec as it is defined. They would have to change their implementation because runc does not comply with the current one.
I can change that PR as you wish, but I'd prefer to do so when either runc or the spec gets respectively fixed or modified.